### PR TITLE
Create basic structure for WhatsApp transport

### PR DESCRIPTION
**Pull request by Hannarst**

### CODE REVIEW

**Comment by Hannarst** [comment]
@hodgestar, @smn 
Ready for review

**Comment by smn** [comment]
:+1:

File: vxyowsup/whatsapp.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    CONFIG_CLASS = WhatsAppTransportConfig
+    transport_type = 'whatsapp'
+
+    def setup_transport(self):
+        config = self.get_static_config()
+        log.info('Transport starting with: %s' % (config.extra_config,))
+        return defer.succeed(1)
+
+    def teardown_transport(self):
+        return defer.succeed(1)
```
Comment (hodgestar) [inline vxyowsup/whatsapp.py]: I think there's no real need to return `defer.succeed(...)` in `setup_transport` and `teardown_transport`?

**Comment by Hannarst** [comment]
Well, I'll remove them anyway when I get to the actual implementation. I'm not sure why they're there in this template

**Pull request merged**